sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md

```diff
@@ -153,26 +153,23 @@ public class AADB2CWebController {
 12. Create a new Java file named *AADB2COidcLoginConfigSample.java* in the *security* folder and open it in a text editor.
 
 13. Enter the following code, then save and close the file:
-<!-- embedme ../azure-spring-boot/src/samples/java/com/azure/spring/b2c/AADB2COidcLoginConfigSample.java#L17-L36 -->
+<!-- embedme ../azure-spring-boot-samples/azure-spring-boot-sample-active-directory-b2c-oidc/src/main/java/com/azure/spring/sample/aad/b2c/security/WebSecurityConfiguration.java#L11-L27 -->
 ```java
 @EnableWebSecurity
-public class AADB2COidcLoginConfigSample extends WebSecurityConfigurerAdapter {
+public class WebSecurityConfiguration extends WebSecurityConfigurerAdapter {
 
     private final AADB2COidcLoginConfigurer configurer;
 
-    public AADB2COidcLoginConfigSample(AADB2COidcLoginConfigurer configurer) {
+    public WebSecurityConfiguration(AADB2COidcLoginConfigurer configurer) {
         this.configurer = configurer;
     }
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        // @formatter:off
         http.authorizeRequests()
-                .antMatchers("/login").permitAll()
-                .anyRequest().authenticated()
-                .and()
+            .anyRequest().authenticated()
+            .and()
```

Comment:
These two lines should be indented. And we could add the `// @formatter:off` here.